db/schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE people
+(
+	ID 		INTEGER PRIMARY KEY AUTOINCREMENT,
+	email	VARCHAR NOT NULL UNIQUE
+);
+
+CREATE TABLE forbiddenPairings
+(
+	ID 			INTEGER PRIMARY KEY AUTOINCREMENT,
+	santa		VARCHAR NOT NULL,
+	assignment	VARCHAR NOT NULL,
+	FOREIGN KEY(santa) REFERENCES people(email)

```

Comment:
Love the foreign keys. But you should really be referencing ids not emails. One good reason: Email is something that might change (for example Ruth was using her college email and one point and switched to gmail after graduating. Ids will basically never change so it's always a good idea to sue them.

Reply:
BTW I realize this means refactoring your algorithm to work with ids instead of emails, but it's better.

Reply:
I have reworked the schema and reset_db.py script so that the table has foreign keys referencing the primary key field of 'people.' Should I still adjust the algorithm to work with ids instead of emails? I don't think it matters?

Reply:
Yes, ids is better. Much less prone to bugs/issues.